contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
+Ids can be specified in three different forms:
+- Id only: `#2000`
+- Repo and id: `zulip#2000`
+- Owner and repo and id `zulip/zulip#2000`
+
+Both the username mention and the id can occur at any time in the message. You
+can also mention multiple ids in a single message.
+
+The bot *requires* a default owner and repo to be configured.
+The configuration file should be located at `~/.contrib_bots/github_detail.ini`.
```

Comment:
I'm not sure about this path, we usually ask for `~/` - why do you want to use a dot-name folder?